docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+    >>> true_positive_rate_group_summary(y_true, y_pred, sensitive_features=sensitive_features)
+    {'overall': 0.5714285714285714, 'by_group': {'a': 0.75, 'b': 0.3333333333333333}}
+    >>> tprp.load_data(X, y_true, sensitive_features=sensitive_features)
+    >>> tprp.gamma(lambda X: y_pred)
+    sign  event    group_id
+    +     label=1  a           0.178571
+                   b          -0.238095
+    -     label=1  a          -0.178571
+                   b           0.238095
+    dtype: float64
+
+.. note::
+
+    In mitigation examples we never need to call :code:`load_data` or
+    :code:`gamma` since the reductions techniques all automatically take care
+    of it. The example merely helps in illustrating the deviation from the
+    corresponding inequalities in :ref:`constraints_binary_classification`.
+
+Similarly to other constraints a ratio-based version is also available:
+
+.. doctest:: mitigation
+
+    >>> tprp = TruePositiveRateParity(ratio_bound=0.9, ratio_bound_slack=0.01)
+    >>> tprp.load_data(X, y_true, sensitive_features=sensitive_features)
+    >>> tprp.gamma(lambda X: y_pred)
+    sign  event    group_id
+    +     label=1  a           0.103571
+                   b          -0.271429
+    -     label=1  a          -0.235714
+                   b           0.180952
+    dtype: float64
+
+.. _equalized_odds:
+    
+Equalized Odds
+^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *Equalized Odds* if it satisfies both
+*True Positive Rate Parity* and *False Positive Rate Parity*, i.e.,
+
+.. math::
+
+    \P[h(X) = y' \given A = a, Y = y] = \P[h(X) = y' \given Y = y] \; \forall a, y, y'
+
+From a usage standpoint this works just like :ref:`true_positive_rate_parity`.
+The constraint violations with :code:`label=1` match exactly the ones from
+True Positive Rate Parity above, while the ones with :code:`label=0` represent
+the violations for False Positive Rate Parity.
+
+.. doctest:: mitigation
+
+    >>> from fairlearn.reductions import EqualizedOdds
+    >>> eo = EqualizedOdds(difference_bound=0.01)
+    >>> eo.load_data(X, y_true, sensitive_features=sensitive_features)
+    >>> eo.gamma(lambda X: y_pred)
+    sign  event    group_id
+    +     label=0  a          -0.333333
+                   b           0.166667
+          label=1  a           0.178571
+                   b          -0.238095
+    -     label=0  a           0.333333
+                   b          -0.166667
+          label=1  a          -0.178571
+                   b           0.238095
+    dtype: float64
+
+.. _error_rate_parity:
+
+Error Rate Parity
+^^^^^^^^^^^^^^^^^
+
+The basic idea behind *Error Rate Parity* is that the error rate should be
+similar between different groups of people.
+
+A classifier :math:`h(X)` satisfies *Error Rate Parity* if it satisfies
+
+.. math::
+
+   \E[\lvert h(X) - Y \rvert \given A = a] - \E[\lvert h(X) - Y \rvert] \leq 0 \; \forall a
```

Comment:
The \rvert and \given symbols are indistinguishable in the rendered equation. And the brackets aren't too different from them either. As a result, this equation is hard to parse.

Reply:
Perhaps split into two - first define an err(X) \def | h(X) - Y | and then use err(X) in the parity definition.

Reply:
I used a more natural definition that makes it clear that this is an error rate.